packages/react-theme-provider/src/ThemeProvider.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { useStylesheet } from '@fluentui/react-stylesheets';
+import { tokensToStyleObject } from './tokensToStyleObject';
+import { ThemeContext } from './ThemeContext';
+import { Theme, ThemePrepared } from './types';
+import { mergeThemes } from './mergeThemes';
+import { useTheme } from './useTheme';
+import * as classes from './ThemeProvider.scss';
+
+/**
+ * Props for the ThemeProvider component.
+ */
+export interface ThemeProviderProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The theme provided by the user.
```

Comment:
nit: Given our own guidelines for documenting props shouldn't this be something like:

```suggestion
   * Defines the theme provided by the user.
```